mesonbuild/mtest.py

```diff
@@ -23,6 +23,7 @@
 import asyncio
 import datetime
 import enum
+from fnmatch import fnmatch
```

Comment:
These types of imports should go beneath `import xxx`. Look at other files for inspiration

Reply:
Or use `fnmatch.fnmatch()`, but indeed our coding style is to have `from x import y` after all `import x`.

Other than this nitpicking, LGTM.

Reply:
If you meant *before*, I did the change.

Reply:
perhaps so, but at least it matches what is in that file!

Reply:
Right, as long as it match existing style, I'm fine :)

Reply:
I don't think we ever properly standardized whether to do it at the top or the bottom. Different files do different things.